networkx/algorithms/community/modularity_max.py

```diff
@@ -287,9 +287,9 @@ def greedy_modularity_communities(
 
     Examples
     --------
-    >>> from networkx.algorithms.community import greedy_modularity_communities
+    >>> greedy_community = nx.community.greedy_modularity_communities
```

Comment:
This is not necessarily a blocker, but I'd argue that this is actually less idiomatic than `from <module> import <function>`.

Reply:
So, in this case @rossbar are you suggesting:
```from networkx.community import greedy_modularity_communities as greedy_community``` 
or maybe
```from networkx.community import greedy_modularity_communities``` and then changing the calls below from `greedy_community` to `greedy_modularity_community`?

Reply:
I think either calling from the top-level namespace or aliasing are equally valid - I get that the main issue is that the function name is long! For the latter, I think importing and aliasing with `as` is preferable (i.e. the way it was before):

```python
from networkx.algorithms.community import greedy_modularity_communities as greedy_communities
```

The former is fine too:

```python
import networkx as nx
nx.community.greedy_modularity_communities(G)
```

This is really just a style nit though - I'm happy to just put it in. Normally I wouldn't even bring it up, but since this is in a docstring example, it's a pattern that is likely to be repeated in user code.

Reply:
The long import followed by `as greedy_communities` reminds me why I like flat namespaces. 
I'd prefer to not have any `import`-like code at the front end and just use `nx.community.greedy_modularity_communities(G)` as your last example suggests. 

Similarly below in this same file for the function `naive_greedy_modularity_community`.